tpdatasrc/tpgamefiles/scr/spell_utils.py

```diff
@@ -421,3 +473,154 @@ def __init__(self, name, argn):
 
     def AddTouchHook(self, hook):
         self.AddHook(ET_OnD20Signal, EK_S_TouchAttack, hook, ())
+
+######## Spell Python Modifier classes ########
+
+### Standard Spell Condition Modifier ###
+
+class SpellPythonModifier(PythonModifier):
```

Comment:
Great!

Reply:
I totally agree, very good idea! Will use it in my code as well. thanks!